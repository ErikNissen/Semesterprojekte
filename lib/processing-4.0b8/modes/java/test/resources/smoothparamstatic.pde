version https://git-lfs.github.com/spec/v1
oid sha256:1962dfb72fb83cd802dc8edf289b55cf75faf6fee7f9f4cfcf97c3dc1d3e3306
size 128
