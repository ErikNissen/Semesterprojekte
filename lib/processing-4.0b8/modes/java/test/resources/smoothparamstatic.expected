version https://git-lfs.github.com/spec/v1
oid sha256:2492f7f65e41e2113eb952c65a5f653c586af34490d7f86781a349c2451aa670
size 961
